client-v2/src/components/FrontsCAPIInterface/SearchInput.tsx

```diff
@@ -131,256 +137,211 @@ class FrontsCAPISearchInput extends React.Component<
-        {this.renderSelectedDates(fromDate, toDate)}
-        <CAPITagInput
-          placeholder={`Type tag name`}
-          onSearchChange={this.handleTagSearchInput}
-          tagsSearchTerm={this.state.searchTerms.tags}
-          onChange={this.handleTypeInput}
-          searchType="tags"
-        />
-        <CAPITagInput
-          placeholder={`Type section name`}
-          onSearchChange={this.handleTagSearchInput}
-          tagsSearchTerm={this.state.searchTerms.sections}
-          onChange={this.handleTypeInput}
-          searchType="sections"
-        />
-
-        <CAPITagInput
-          placeholder={`Type commissioning desk name`}
-          onSearchChange={this.handleTagSearchInput}
-          tagsSearchTerm={this.state.searchTerms.desks}
-          onChange={this.handleTypeInput}
-          searchType="desks"
-        />
-        <CAPIFieldFilter
-          placeholder={'Select one or more'}
-          filterTitle="star rating for reviews"
-          filterType="ratings"
-          items={[
-            { value: '1', id: '1 Star' },
-            { value: '2', id: '2 Stars' },
-            { value: '3', id: '3 Stars' },
-            { value: '4', id: '4 Stars' },
-            { value: '5', id: '5 Stars' }
-          ]}
-          onChange={this.handleDropdownInput}
-        />
-        <CAPIDateRangeInput
-          start={this.state.fromDate}
-          end={this.state.toDate}
-          onDateChange={this.onDateChange}
-        />
-      </React.Fragment>
+      !!this.state.tags.length ||
+      !!this.state.sections.length ||
+      !!this.state.desks.length ||
+      !!this.state.ratings.length ||
+      !!this.state.query ||
+      !!this.state.fromDate ||
+      !!this.state.toDate
     );
   }
+
+  private get shouldShowDate() {
+    return this.state.fromDate || this.state.toDate;
+  }
+
+  private addUniqueStringToStateKey(key: keyof StringArrSearchItems) {
+    return ({ id }: { id: string }) => {
+      const arr = this.state[key];
+      this.setStateAndRunSearch({
+        ...this.state,
+        [key]: arr.includes(id) ? arr : [...arr, id]
+      });
+    };
+  }
+
+  private removeStringFromStateKey(
+    key: keyof StringArrSearchItems,
+    val: string
+  ) {
+    const arr = this.state[key];
+    this.setStateAndRunSearch({
+      ...this.state,
+      [key]: arr.includes(val) ? arr.filter(v => v !== val) : arr
+    });
+  }
+
+  private debouncedRunSearchAndRestartPolling = () => {};
+
+  private setStateAndRunSearch<K extends keyof FrontsCAPISearchInputState>(
+    state: Pick<FrontsCAPISearchInputState, K>
+  ) {
+    this.setState(state, () => {
+      this.debouncedRunSearchAndRestartPolling();
+    });
+  }
 }
 
-export { AsyncState, CAPISearchQueryReponse };
+const mapDispatchToProps = (dispatch: Dispatch) => ({
+  fetchFeed: (query: string, liveParams: object, previewParams: object) => {
+    const maybeArticleId = getIdFromURL(query);
+    dispatch(fetchLive(liveParams, !!maybeArticleId));
```

Comment:
This is where we fetch both at the same time. If there were a sub-component that handled some of this stuff then this wouldn't be necessary and we could handle each feed separately but again as mentioned in the PR. Happy to go with whatever people think makes sense.